README.md

```diff
@@ -306,6 +307,19 @@ service:
       exporters: [opencensus, zipkin]
 ```
 
+### <a name="config-other"></a>Other
+
+Beyond standard YAML configuration of the Collector, the Collector also
+respects additional configuration parameters such as certain environment
+variables. This section captures these additionally respected configuration
+options.
+
+**HTTP_PROXY / HTTPS_PROXY**
+
+The Collector respects the `HTTP_PROXY` and `HTTPS_PROXY` environment
+variables. If set at Collector start time then both REST and gRPC exporters
+will proxy traffic as defined by these environment variables.
+
```

Comment:
We should also mention `NO_PROXY` which can take a comma separated list of endpoints that should not be router through the specified proxy.

Reply:
I think we should document this per exporter. The env vars are supported by the net/http standard package but it is possible that some future exporters might not rely on it. Alternately, we can document this here and mandate that every exporter supports these env variables if applicable.

Reply:
Good call, should be fixed now